src/savefile/savefile.c

```diff
@@ -26,13 +25,11 @@ extern OSMesgQueue dmaMessageQ;
 #define SRAM_pageSize    0xd 
 #define SRAM_relDuration 0x2
 
-#define SRAM_CHUNK_DELAY        OS_USEC_TO_CYCLES(10 * 1000)
+#define SRAM_CHUNK_DELAY (10 * 1000)
```

Comment:
Minor: can you include the units in the name (i.e., `SRAM_CHUNK_DELAY_USECS`)? Makes it unambiguous to readers.